lib/bloc/account/account_bloc.dart

```diff
@@ -74,9 +74,10 @@ class AccountBloc extends Cubit<AccountState> with HydratedMixin {
 
   Future _startRegisteredNode() async {    
     final credentials = await _credentialsManager.restoreCredentials();
+    final seed = bip39.mnemonicToSeed(credentials.mnemonic);
```

Comment:
We need to make sure that we all create new wallets since this is not backward compatible. It is reasonable because we are pre alpha.

Reply:
Correct.

I tested it with new wallets, as well as restore previous wallet.

Just upgrading c-breez "in place" will not work. It will probably need the previous app to be uninstalled first.